aurora/scripts/utils.js

```diff
@@ -168,6 +172,45 @@ async function forceIncreaseBalance(signer, config, fastBridgeAddress, token, re
   console.log("Transaction hash: ", receipt.hash);
 }
 
+async function get_pending_transfers(config, nonce) {
+    const nearAPI = require("near-api-js");
+
+    const { connect } = nearAPI;
+
+    let connectionConfig = {
```

Comment:
The RPC URLs should be included in the config files

Reply:
fixed: https://github.com/aurora-is-near/fast-bridge-protocol/pull/130/commits/64a50b71205c844681306d55fd96a378f1addddb